wordpress-mobile/WordPress-FluxC-Android
# Quick Start for Existing Users V2 - Refactor code to support different quick start types (FluxC)

Parent https://github.com/wordpress-mobile/WordPress-Android/issues/16350

This PR splits `QuickStartTask` into two different sets of tasks for different quick start types. 
PS: Currently, no new tasks are added to the "existing site"  quick start type.

## How to test?

See corresponding WPAndroid PR: https://github.com/wordpress-mobile/WordPress-Android/pull/16500
